charts/backstage/values.yaml

```
@@ -79,6 +79,8 @@ ingress:
 # -- Backstage parameters
 # @default -- See below
 backstage:
+  # -- Number of deployment replicas
+  replicas: 1
```

Just a formatting nit, so we're consistent with the rest of the file.

```suggestion

  # -- Number of deployment replicas
  replicas: 1

```